### PR TITLE
feat(EditPolicy): RHICOMPL-1421 Filter systems by OS major

### PR DESCRIPTION
**Pull request by akofink**
When editing a policy, only show systems with a corresponding OS major
version to the policy benchmark

Signed-off-by: Andrew Kofink <akofink@redhat.com>

## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [X] Input Validation
- [X] Output Encoding
- [X] Authentication and Password Management
- [X] Session Management
- [X] Access Control
- [X] Cryptographic Practices
- [X] Error Handling and Logging
- [X] Data Protection
- [X] Communication Security
- [X] System Configuration
- [X] Database Security
- [X] File Management
- [X] Memory Management
- [X] General Coding Practices

### CODE REVIEW

File: src/SmartComponents/EditPolicy/__snapshots__/EditPolicy.test.js.snap [hunk 1]
```diff
@@ -102,6 +102,7 @@ exports[`EditPolicy expect to render with active tab open 1`] = `
             ]
           }
           compact={true}
+          defaultFilter="os_major_version = undefined"
```
Comment (vkrizan) [inline src/SmartComponents/EditPolicy/__snapshots__/EditPolicy.test.js.snap:105]: Can you please update fixtures to prevent `undefined` getting here?

**Review by vkrizan — COMMENTED** [review]
Nice. I thought that it would need more changes.

File: src/SmartComponents/EditPolicy/__snapshots__/EditPolicy.test.js.snap (hunk 1 above)
Comment (akofink) [reply src/SmartComponents/EditPolicy/__snapshots__/EditPolicy.test.js.snap:106]: :see_no_evil: I always forget to update props in tests. Thanks for pointing it out. Updated!

**Review by vkrizan — APPROVED** [review]
ty

**Pull request merged**